lightning/config/config.go

```diff
@@ -504,7 +504,7 @@ func (cfg *Config) Adjust() error {
 
 	// handle mydumper
 	if cfg.Mydumper.BatchSize <= 0 {
-		cfg.Mydumper.BatchSize = 100 * _G
+		cfg.Mydumper.BatchSize = 10 * _G
```

Comment:
keep the default as 100 GB when the backend is "importer".

Reply:
I have test 1T data with importer backend with 10G as the default config, seems there is no obvious impact.